__init__.py

```diff
@@ -1150,6 +1151,38 @@ def _speak_weather(self, dialog):
         self.log.info("Speaking dialog: " + dialog.name)
         self.speak_dialog(dialog.name, dialog.data, wait=True)
 
+    @skill_api_method
+    def get_current_weather(self, message=None):
+        """Get the current temperature and condition code.
+        
+        Args:
+            message (Message): [Optional] mock message will be parsed to
+                               determine weather intent.
+
+        Returns:
+            Dict: {
+                temperature: current temperature
+                high_temperature: forecasted high for today
+                low_temperature: forecasted low for today
+                condition_code: code representing overall weather condition
+                                see Maps for all codes in skill/weather.py
+            }
+        """
+        self.log.debug("Requested current weather information:")
+        if message is None:
+            message = Message("", data={"utterance": ""})
+        intent_data = self._get_intent_data(message)
+        weather = self._get_weather(intent_data)
+        data = {
```

Comment:
This is a nitpick, I admit, but the rest of the code uses the dict() constructor to build dictionaries with literal keys.  Why do it differently here?

Reply:
counter nitpick, change the rest of the codebase to also use the de facto syntax?

Reply:
I'm just used to this style and hence find it more readable, but not a conscious decision. Changing in favor of consistency with the code in this Skill.